apm-agent/src/test/resources/configuration.asciidoc.ftl

```diff
@@ -101,7 +101,7 @@ Click on a key to get more information.
 
 <#list config as category, options>
 [[config-${category?lower_case?replace(" ", "-")}]]
-=== ${category} configuration options
+=== ${category}
```

Comment:
This suggestion will abbreviate the TOC to just the `category` while ensuring the page headings on each individual page still communicate that these are configuration options.

```suggestion
=== ${category} configuration options

++++
<titleabbrev>${category} </titleabbrev>
++++

```

Reply:
I made a screencast, but GitHub won't let me upload it. Give this a shot locally and this should resolve your concerns in your previous comment.